packages-ts/starknet-gauntlet-ledger/src/index.ts

```diff
@@ -76,6 +77,16 @@ class LedgerSigner implements SignerInterface {
     return this.sign(msgHash)
   }
 
+  async signDeployAccountTransaction(transaction: DeployAccountSignerDetails): Promise<Signature> {
+    // TODO: implement this
+    return []
+  }
+
+  async signDeclareTransaction(transaction: DeclareSignerDetails): Promise<Signature> {
+    // TODO: implement this
+    return []
+  }
```

Comment:
@sdrug A couple of TODOs here since I couldn't find an upstream implementation in the ledger repo. Perhaps we can base it on `starknet.js` internals

Reply:
Noted 👍